reactnative/RNPlayground/app/i18n/en.ts

```diff
@@ -10,6 +10,16 @@ const en = {
     readyForLaunch: "Your app, almost ready for launch!",
     exciting: "(ohh, this is exciting!)",
   },
+  aboutScreen: {
```

Comment:
So, this is where you place your copies? , in case we want to have a version of the app in spanish, would this be then a different file?

Reply:
Yep `EN` translations in this file, with more languages available in the `i18n` folder, including Spanish and Chinese.

`i18next` is a internationalization framework, which will auto translate dpending on the locale of the user. More info here; https://react.i18next.com/